### PR TITLE
Fix: Remove the close button from the screen

### PR DESCRIPTION
**Pull request by 30mar**
## Description
The close button is not working is the "Warn others" screen.

**UPDATE**: in the comments it was stated that the updated design in figma doesn't have the button so it should be removed from the app as well.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4181

## Screenshot
IMG_0005
IMG_0006

### CODE REVIEW

**Review by marcussc — APPROVED** [review]

**Review by tcfos — APPROVED** [review]

**Comment by KaiTeuber** [comment]
But you still can pull it down to close the screen?

**Comment by 30mar** [comment]
@KaiTeuber no you have to go back to dismiss by pull-down, but that is the expected behavior right ?

**Comment by KaiTeuber** [comment]
>  but that is the expected behavior right ?
I don't know. But after the change you still can pull the screen down and viewmodel.dissmis() is called - I think

**Review by nickguendling — CHANGES_REQUESTED** [review]
There is a hack in `ExposureSubmissionTestResultConsentViewModel` that makes this change also change the text. See line 63-70.

**Comment by marcussc** [comment]
> There is a hack in `ExposureSubmissionTestResultConsentViewModel` that makes this change also change the text. See line 63-70.

@whiskey Maybe you can help out here? :)

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift [hunk 1]
```diff
@@ -181,12 +181,7 @@ class ExposureSubmissionCoordinator: NSObject, ExposureSubmissionCoordinating, R
 				self?.model.exposureSubmissionService.loadSupportedCountries(
 					isLoading: isLoading,
 					onSuccess: { supportedCountries in
-						self?.showTestResultSubmissionConsentScreen(
-							supportedCountries: supportedCountries,
-							onDismiss: {
-								self?.showTestResultAvailableCloseAlert()
-							}
-						)
+						self?.showTestResultSubmissionConsentScreen(supportedCountries: supportedCountries, onDismiss: nil)
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:184]: please keep the line break style
```suggestion
self?.showTestResultSubmissionConsentScreen(
 							supportedCountries: supportedCountries,
 							onDismiss: nil
 						)						self?.showTestResultSubmissionConsentScreen(supportedCountries: supportedCountries, onDismiss: nil)
```

**Review by KaiTeuber — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift (hunk 1 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:184]: Ok sure

**Review by KaiTeuber — APPROVED** [review]

**Review by nickguendling — APPROVED** [review]

**Pull request merged**